example.py

```diff
@@ -2,6 +2,7 @@
 from oxo_tourney.player import RandomPlayer
 from oxo_tourney.tourney import Tourney
 
+# 
```

Comment:
Was this the only change? Is it required?

Reply:
Hi Grahame, this was me testing github out with a quick change which needs to be removed

Reply:
Deleted "#" and change should commit to the david-ai-implementation and come through to the pull request